NOMIS/nomis.go

```diff
@@ -51,25 +51,32 @@ type NameDetails struct {
 }
 
 var (
-	censusNationalStatistic = false
+	censusNationalStatistic = true
 	fileName                string
 	fullURLFile             string
 	title                   string
-	metaTitle               []int
 	num                     int
 )
 
 const (
-	censusYear    string = "2011"
-	censusVersion string = "1"
+	censusYear         string = "2011"
+	censusVersion      string = "1"
+	censusPersonalData string = "Sometimes we need to make changes to data if it is possible to identify individuals. This is known as 'statistical disclosure control'. In the 2011 Census, we:\u000a\u000a" +
+
+		"- swapped records (targeted record swapping), for example if a household could be identified because it has unusual characteristics, the record was swapped with a similar one from the same local area" +
+		"(in specific cases the household was swapped with one in a nearby local authority) \u000a" +
+		"- reduced the detail included for areas where fewer people lived and could be identified, such as electoral wards\u000a\u000a" +
+
+		"Read more about these [methods and why we chose them for the 2011 Census.]" +
+		"(https://webarchive.nationalarchives.gov.uk/20160129174312/http:/www.ons.gov.uk/ons/guide-method/census/2011/the-2011-census/processing-the-information/statistical-methodology/statistical-disclosure-control-for-2011-census.pdf)"
 )
 
 //CensusContactDetails returns the default values for contact details
 func CensusContactDetails() models.ContactDetails {
 	return models.ContactDetails{
-		Email:     "support@nomisweb.co.uk",
+		Email:     "Census.customerservices@ons.gov.uk",
 		Name:      "Nomis",
-		Telephone: "+44(0) 191 3342680",
+		Telephone: "01329 444972",
```

Comment:
The telephone number should also match the rest of the website with `+44 (0)` instead of the initial `0` I believe.